prefect_airbyte/connections.py

```diff
@@ -45,21 +44,28 @@ async def trigger_sync(
     when it receives an error status code from an API call.
 
     Args:
-        str airbyte_server_host : Hostname of Airbyte server where connection is
-            configured.
-        str airbyte_server_port: Port that the Airbyte server is listening on.
-
-        str airbyte_api_version: Version of Airbyte API to use to trigger connection
-            sync.
-        str connection_id: the Airbyte connection ID
-        int poll_interval_s: how often to poll the
-            Airbyte API for sync status, if provided this will
-            override the default polling time of 15 seconds.
-        bool status_updates: whether to log status as the task polls jobs
-        str timeout: The request `timeout` for the `httpx.AsyncClient`
+        connection_id (str): the Airbyte connection ID to trigger a sync for.
+        airbyte_server_host (str, optional): Hostname of Airbyte server where the
+            connection is configured. Defaults to "localhost".
+        airbyte_server_port (int, optional): Port where Airbyte instance is listening.
+            Defaults to "8000".
+        airbyte_api_version (str, optional): Version of Airbyte API to use to trigger
+            connection sync. Defaults to "v1".
+        poll_interval_s (int, optional): how often to poll the
+            Airbyte API for sync status. Defaults to 15 seconds.
+        status_updates (bool, optional): whether to log sync job status while polling.
+            Defaults to False.
+        timeout (int, optional): The POST request `timeout` for the `httpx.AsyncClient`.
+            Defaults to 5.
+
+    Raises:
+        ValueError: if `connection_id` is not a valid UUID
+        err.AirbyteSyncJobFailed: if airbyte returns `JOB_STATUS_FAILED`
+        err.AirbyteConnectionInactiveException: if a specified connection is inactive
+        err.AirbyeConnectionDeprecatedException: if a specified connection is deprecated
 
     Returns:
-        dict: connection_id (str) and succeeded_at (timestamp str)
+        dict: job metadata, including the connection ID and final status
```

Comment:
Same comment about type in the docstrings here.
```suggestion
        Job metadata, including the connection ID and final status
```